### DISCUSSION TITLE
Dagstermill dependencies

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18277] [Q&A] [answered]
Hey Guys am trying to use dagster mill , question is , are there dependencies i need ti have installed before ?
I get errors of cmake not being available

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1700981979847279?thread_ts=1700981979.847279&cid=C01U954MEER).*

**Comment by smackesey** [comment, marked as answer]
`dagstermill` doesn't directly require any dependencies not already included in its setup.py (and therefore automatically installed by `pip install dagstermill`), but it is possible some nth-order dependencies require non-Python dependencies like `cmake`. I would just install `cmake` and anything else it complains about and then see if it works.
